deno/compress_image/src/mod.ts

```diff
@@ -0,0 +1,77 @@
+import { base64Encode } from "./deps.ts";
+
+export default async function (req: any, res: any) {
+  const { provider, image } = JSON.parse(req.payload);
```

Comment:
Would you please add some error handling in case payload isn't a valid JSON string?